src/style.scss

```diff
@@ -49,7 +49,7 @@
 @media (hover: hover) {
   .trigger {
     &:hover {
-      background-color: var(--auro-color-ui-bkg-hover-on-light);
+      background-color: var(--auro-color-background-lightest);
```

Comment:
We should apply this background-color as the default for `.trigger` and then there should be no need to apply it here on hover as it won't be changing.